cvelib/cli.py

```diff
@@ -238,13 +239,15 @@ def cli(
 @click.pass_context
 @handle_cve_api_error
 def publish(ctx: click.Context, cve_id: str, cve_json_str: str, print_raw: bool) -> None:
-    """Publish a CVE record for an already-reserved CVE ID.
+    """Publish a CVE record for a reserved (or rejected) CVE ID.
 
-    Will update if the CVE record already exists.
+    If the CVE is already published, this action will update its record. A published CVE can only be
+    moved to the rejected state with an appropriate reject record (see `cve reject`).
+
+    Example:
```

Comment:
Maybe be more explicit that after publishing a record for a CVE ID, the CVE ID cannot go back to the reserved state.

Reply:
Sure, added!